### ISSUE TITLE
[iOS] Can't upgrade to 6.11.0

### ISSUE THREAD

**Issue by ohjeyong** [react-native-community/react-native-webview#825]
**Bug description:**

Version 6.9.0 works fine and great, but after upgrading 6.11.0, iOS build is successful but as soon as launch the app, it crashed.
I downgrade to 6.9.0, it works fine. I remove node_modules, yarn.lock, Pods, Podfile.lock and upgrade to 6.11.0, also it crashed. I have reproduced it twice and result is same.

**To Reproduce:**

Upgrade from 6.9.0 to 6.11.0

**Expected behavior:**

Works fine.

**Screenshots/Videos:**

This is error log.
```
2019-08-30 11:46:31.515143+0900 service_app[1969:1069927] -[UIViewController setFrame:]: unrecognized selector sent to instance 0x107d24410
2019-08-30 11:46:31.516546+0900 service_app[1969:1069927] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[UIViewController setFrame:]: unrecognized selector sent to instance 0x107d24410'
```

**Environment:**
 - OS: Mac Mojave
 - OS version: 10.14.6
 - react-native version: 0.60.5
 - react-native-webview version: 6.11.0

**Comment by ouabing** [comment]
I'm facing a different issue after upgrading to 6.11.0, videos won't play any more.

**Comment by Titozzz** [comment]
Going to check this ASAP

**Comment by Titozzz** [comment]
Injected javascript was broken in 6.11.0, fix is deploying right now, any other issue ?

**Comment by Titozzz** [comment]
closed in 6.11.1
